jenkins-jobcfg.py

```diff
@@ -107,7 +107,7 @@ def jenkins_check_config(config):
     assert response.status_code == 200
 
 def jenkins_configs(config_file):
-    all_configs = os.path.isfile(config_file) and yaml.load(
+    all_configs = os.path.isfile(config_file) and yaml.safe_load(
```

Comment:
What if i comment on this but change it afterward?

Reply:
Well it can be recovered, the comment is marked outdated in the conversations (this is a dynamic feature).